src/lib/viewers/controls/annotations/annotations-button-region-tooltips/PersistentOnboardingBoxEditAnnotationsTooltip.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, PropsWithChildren } from 'react';
+import classNames from 'classnames';
+
+// TODO: Compare bundle sizes and possibly re-write here
+import { TargetedClickThroughGuideTooltip } from 'box-ui-elements/es/features/targeting';
+
+import { TargetingApi } from '../../../../types';
+
+import { ControlsLayerContext } from '../../controls-layer';
+import { DependenciesContext } from '../../dependencies';
+
+import './PersistentOnboardingBoxEditAnnotationsTooltip.scss';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function PersistentOnboardingBoxEditAnnotationsTooltip({
+    children,
+    isEnabled = false,
+}: Props): JSX.Element {
+    const { experiences, isLocaleLanguageEnglish, onPersistentOnboardingTooltipBack } = useContext(DependenciesContext);
+    const { setIsForced } = useContext(ControlsLayerContext);
+
+    const { persistentOnboardingBoxEditAnnotations: persistentOnboardingBoxEditAnnotationsExperience } = experiences;
+    const useTargetingApi = (): TargetingApi => ({
+        ...persistentOnboardingBoxEditAnnotationsExperience,
+        onClose: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onClose();
+            setIsForced(false);
+        },
+        onComplete: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onComplete();
+            setIsForced(false);
+        },
+        onShow: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onShow();
+            setIsForced(true);
+        },
+    });
+
+    const title = __('persistent_onboarding_box_edit_annotations_tooltip_title');
+    const body = __('persistent_onboarding_box_edit_annotations_tooltip_body');
+
+    const image = (
+        <div
+            className={classNames('bp-PersistentOnboardingBoxEditAnnotationsTooltip-image', {
+                isLocalized: isLocaleLanguageEnglish,
```

Comment:
Should this condition be inverted?

Reply:
Good catch. I have actually already fixed that. Am resolving.